app/javascript/components/explore/DifferentialExpressionPanel.jsx

```diff
@@ -41,13 +43,32 @@ export default function DifferentialExpressionPanel({
 
   const [checked, setChecked] = useState(initChecked(deGenes))
   const [deFileUrl, setDeFileUrl] = useState(null)
+  const [showDeModal, setShowDeModal] = useState(false)
 
   /** Check radio button such that changing group unchecks all buttons */
   function changeRadio(event) {
     const newChecked = initChecked(deGenes, event.target.value)
     setChecked(newChecked)
   }
 
+  const deModalContent = (
+    <div>
+      <p>These are exploratory results calculated automatically by SCP, not the study owner. Any discrepancies between
+        these results and an associated publication may be because it benefits from different methods than the ones that
+        SCP uses. They are intended purely as an aid in exploring this dataset.&nbsp;
```

Comment:
Trim, and clarify ambiguous pronouns (it = publication, they = these results):

```suggestion
        these results and an associated publication may be because it benefits from methods different from SCP's. These results are intended purely as an aid in exploring this dataset.&nbsp;
```